arionBackend/transformation/esper/esper_transformator.py

```diff
@@ -0,0 +1,6 @@
+from arionBackend.transformation.transformator import Transformer
+
+
+class EsperTransformer(Transformer):

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)